KarhooUISDK/Screens/QuoteList/QuoteListFilters/QuoteListFiltersViewController.swift

```diff
@@ -0,0 +1,206 @@
+        $0.addTarget(self, action: #selector(closePressed), for: .touchUpInside)
+    }
+    private lazy var resetButton = UIButton().then {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.setTitle("reset_filter", for: .normal)
+        $0.setTitleColor(KarhooUI.colors.accent, for: .normal)
+        $0.titleLabel?.font = KarhooUI.fonts.bodySemibold()
+        $0.addTarget(self, action: #selector(resetPressed), for: .touchUpInside)
+    }
+    private lazy var footerView = UIView().then {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.backgroundColor = KarhooUI.colors.background1
+        $0.addShadow()
+    }
+    private lazy var confirmButton = MainActionButton().then {
+        $0.setTitle(UITexts.Generic.save.uppercased(), for: .normal)
+        $0.addTarget(self, action: #selector(savePressed), for: .touchUpInside)
+    }
+
+    // MARK: - Lifecycle
+
+    init() {
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+
+    override func loadView() {
+        setupView()
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        assert(presenter != nil, "Presented needs to be assinged using `setupBinding` method")
+        presenter.viewDidLoad()
+    }
+
+    override func viewWillAppear(_ animate: Bool) {
+        super.viewWillAppear(animate)
+        presenter.viewWillAppear()
+        updateConfirmButtonTitle()
+    }
+
+    // MARK: - Setup business logic
+
+    func setupBinding(_ presenter: QuoteListFiltersPresenter) {
+        self.presenter = presenter
+    }
+
+    // MARK: - Setup view
+
+    private func setupView() {
+        setupProperties()
+        setupHierarchy()
+        setupLayout()
+    }
+
+    private func setupProperties() {
+        view = UIView()
+        view.backgroundColor = KarhooUI.colors.background1
+    }
+
+    private func setupHierarchy() {
+        view.addSubview(headerStackView)
+        view.addSubview(resetButton)
+        view.addSubview(scrollView)
+        view.addSubview(footerView)
+        scrollView.addSubview(stackView)
+        headerStackView.addArrangedSubviews([
+            headerLabel,
+            closeButton
+        ])
+        stackView.addArrangedSubviews([
+            SeparatorView(fixedHeight: 1200, color: .blue),
+            SeparatorView()
+        ])
+        footerView.addSubview(confirmButton)
+    }
+
+    private func setupLayout() {
+        scrollView.anchor(
+            top: resetButton.bottomAnchor,
+            left: view.leftAnchor,
+            right: view.rightAnchor,
+            bottom: footerView.topAnchor,
+            paddingTop: UIConstants.Spacing.standard
+        )
+        stackView.anchor(
+            top: scrollView.topAnchor,
+            left: scrollView.leftAnchor,
+            right: scrollView.rightAnchor, bottom: scrollView.bottomAnchor,
```

Comment:
`bottom` needs to be on a dedicated row. Can be done here or in a future PR for a task that is currently in progress